### ISSUE TITLE
Explaination of pose calculation from logits of PCAE encoder

### ISSUE THREAD

**Issue by sakshamjindal** [akosiorek/stacked_capsule_autoencoders#11]
Hi,

Could you please explain how you came with the affine transform matrix from pose logits (1x6) from the output of PCAE encoder (function [here](https://github.com/phanideepgampa/stacked-capsule-networks/blob/dcc5279630993b0f240b91669820a179a299f5df/model.py#L83)). It would be really helpful if you could point to relevant source ? 

Also, I was wondering what could be way of working around of calculating 3D pose if we had pose logits for objects in three dimension space (say 3x3 rotation matrix or quarternions) ? 

Thanks
Saksham
